eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -90,13 +90,6 @@ jobs:
 
       - template: ../steps/cache-maven-repository.yml
 
-      - template: /eng/common/pipelines/templates/steps/replace-relative-links.yml
```

Comment:
We don't want to remove this yet until we fix all the relative links in the repo.

Reply:
Missed the step. Will revert it back for all lang repo